bin/nexus-disable-keycloak-realm

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Hewlett Packard Enterprise Development LP
+
+command -v jq >/dev/null 2>&1 || { echo >&2 "command not found: jq"; exit 1; }
+
+. "$(dirname "$0")/nexus-ready"
+
+echo -n >&2 "Disable Keycloak Authenticating security realm..."
+
```

Comment:
Does this need to be idempotent?  What is the effect in the case where the item is missing?

Reply:
If the Keycloak realm isn't active, this action is effectively a no-op. It downloads the set of active realms, there is nothing to delete, and then puts back the same set of active realms. You can verify this behavior by running `hack/nexus-teardown` repeatedly after having run `hack/nexus-create` and `hack/nexus-init`.